hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/client/SparkRDDWriteClient.java

```diff
@@ -59,14 +62,19 @@
 
 import java.util.List;
 import java.util.Map;
+import java.util.concurrent.atomic.AtomicLong;
 import java.util.function.BiConsumer;
+import java.util.stream.Collectors;
 
 @SuppressWarnings("checkstyle:LineLength")
 public class SparkRDDWriteClient<T> extends
     BaseHoodieWriteClient<T, JavaRDD<HoodieRecord<T>>, JavaRDD<HoodieKey>, JavaRDD<WriteStatus>> {
 
   private static final Logger LOG = LoggerFactory.getLogger(SparkRDDWriteClient.class);
 
+  // true if current write client is representing metadata table.
+  private final boolean isMetadataTable;
```

Comment:
the flag is not used anywhere.

Reply:
This would be used in subsequent PRs. This is for specific handling of write status based on whether it belongs to data table or metadata table.

Reply:
remove it in this PR.

Reply:
done.